docs/how-to-guides/web-portal/web-portal-24-04-or-later/use-security-profiles.md

```diff
@@ -0,0 +1,74 @@
+(how-to-web-portal-use-security-profiles)=
+# How to use security profiles
+
+You can use security profiles to automatically audit and fix your managed instances on a schedule using the [Ubuntu Security Guide (USG)](https://documentation.ubuntu.com/security/docs/compliance/usg/).
+
+```{note}
+This feature is only available in self-hosted **Landscape 25.04** and later.
+```
+
+## Create a security profile
+
+On each instance you want to manage, follow the steps in [Install Ubuntu Security Guide](https://documentation.ubuntu.com/security/docs/compliance/usg/install-usg/), then:
```

Comment:
```suggestion
On each instance you want to manage, follow the steps in [Ubuntu Security's installation guide](https://documentation.ubuntu.com/security/docs/compliance/usg/install-usg/), then:
```
(just explaining why I made this suggestion) When I link out to other product documentation, I try to make this as clear as possible in the link text so users know they're leaving the Landscape docs.

The wording is a bit awkward though, so feel free to change if you have something better in mind. Could also be like "Ubuntu Security | Install USG" or "Ubuntu Security's documentation on installing USG"

Reply:
I'm okay with the wording as-is. Thanks!